test/crossfilter-test.js

```diff
@@ -177,6 +176,46 @@ suite.addBatch({
       }
     },
 
+    "up to 64 dimensions supported": function() {
+      var data = crossfilter([]);
+      for (var i = 0; i < 64; i++) data.dimension(function() { return 0; });
+    },
+
+    "can add and remove 64 dimensions repeatedly": function() {
+      var data = crossfilter([]),
+          dimensions = [];
+      for (var j = 0; j < 10; j++) {
+        for (var i = 0; i < 64; i++) dimensions.push(data.dimension(function() { return 0; }));
+        while (dimensions.length) dimensions.pop().remove();
+      }
+    },
+
+    "filtering with more than 32 dimensions": function() {
+      var data = crossfilter();
+      var dims = {};
+
+      for (var i = 0; i < 50; i++) {
+        data.add([{value: i}]);
+      }
+
+      var dimfunc = function(i) {
+        return function(val) {
+          return val.value == i;
+        }
+      }
```

Comment:
That would be great
On Fri, Aug 7, 2015 at 11:17 AM Gordon Woodhull <notifications@github.com>
wrote:

> In test/crossfilter-test.js
> <https://github.com/crossfilter/crossfilter/pull/17#discussion_r36539820>:
>
> > +      }
> > +    },
> > +
> > +    "filtering with more than 32 dimensions": function() {
> > +      var data = crossfilter();
> > +      var dims = {};
> > +
> > +      for (var i = 0; i < 50; i++) {
> > +        data.add([{value: i}]);
> > +      }
> > +
> > +      var dimfunc = function(i) {
> > +        return function(val) {
> > +          return val.value == i;
> > +        }
> > +      }
>
> I see. That's a good basic test. I wonder if I can devise a test with
> synthetic data that is more thorough.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/crossfilter/crossfilter/pull/17/files#r36539820>.
>